client-js/app/elements/labrad-registry.ts

```diff
@@ -132,6 +221,8 @@ export class LabradRegistry extends polymer.Base {
    * that contain substring in filterText
    */
   filterFunc(item) {
+    const selectedIdx = (this.$.search.value) ? 0 : this.getDefaultSelectedItem();
+    this.set('selectedIdx', selectedIdx);
```

Comment:
I already have a branch ready for #278 that eliminates this issue and removes all computed properties in the process.